Components/Metrics/PointToSurfaceDistance/elxPointToSurfaceDistanceMetric.h

```diff
@@ -0,0 +1,172 @@
+/*======================================================================
+
+  This file is part of the elastix software.
+
+  Copyright (c) University Medical Center Utrecht. All rights reserved.
+  See src/CopyrightElastix.txt or http://elastix.isi.uu.nl/legal.php for
+  details.
+
+     This software is distributed WITHOUT ANY WARRANTY; without even
+     the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+     PURPOSE. See the above copyright notices for more information.
+
+======================================================================*/
+#ifndef __elxPointToSurfaceDistanceMetric_H__
+#define __elxPointToSurfaceDistanceMetric_H__
+
+#include "elxIncludes.h" // include first to avoid MSVS warning
+#include "itkPointToSurfaceDistanceMetric.h"
+
+namespace elastix
+{
+
+/**
+ * \class TransformRigidityPenalty
+ * \brief A penalty term based on non-rigidity.
```

Comment:
Is this indeed a penalty term based on non-rigidity?

Reply:
These lines have been removed.